source/element-parameters.md

```diff
@@ -32,39 +32,39 @@ Any group can be given a **name** and the values can be used in another group of
 using **import**.
 For example:
 ```{code} yaml
-ApertureP:
-  name: ap1
+ap1:
+  kind: ApertureP
```

Comment:
Hmmm. I'm rethinking giving names to parameter groups. As an alternative, parameter groups could be identified using a `<element-name>.<parameter-group-name>` syntax. For example, if an  `ApertureP` group is used in an element named `Q1` then the group could be referred to as `Q1.ApertureP`. We should discuss this at the next meeting.

Reply:
To implement what you are suggesting, we could remove the first two ap1/ap2 examples and just stay with the q1/q2 example?

Reply:
I pushed a change now that adds examples for both.